src/python/WMCore/WMSpec/Steps/Executors/LogArchive.py

```diff
@@ -127,7 +143,25 @@ def execute(self, emulator=None, **overrides):
                     'GUID': None
                     }
 
+        eosFileInfo = {'LFN': self.getEOSLogLFN(),
+                       'PFN': tarBallLocation,
+                       'PNN': None,
+                       'GUID': None
+                       }
         signal.signal(signal.SIGALRM, alarmHandler)
+        signal.alarm(waitTime)
+        try:
+            eosmanager(eosFileInfo)
+            eosServerPrefix = eosStageOutParams['lfn-prefix'].replace("root://eoscms.cern.ch//eos/cms",
+                                                                      "https://eoscmshttp.cern.ch")
+            self.report.setLogURL(eosServerPrefix + eosFileInfo['LFN'])
+        except Alarm:
+            msg = "Indefinite hang during stageOut of logArchive to EOS, ignoring it"
+            logging.exception(msg)
+        except Exception as ex:
+            logging.exception("EOS copy failes lfn: %s" % eosFileInfo['LFN'])
```

Comment:
How about `logging.exception("EOS copy failed for lfn: %s. Error %s", eosFileInfo['LFN'], str(ex))`?